lucid-book-store/public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
```

Comment:
Give good title name

Reply:
Okay I will raise PR for smaller changes, I will also change the title